dot/parachain/overseer/types.go

```diff
@@ -14,7 +14,7 @@ type Subsystem interface {
 	// Run runs the subsystem.
 	Run(ctx context.Context, OverseerToSubSystem chan any, SubSystemToOverseer chan any)
 	Name() parachaintypes.SubSystemName
-	ProcessActiveLeavesUpdateSignal()
+	ProcessActiveLeavesUpdateSignal(parachaintypes.ActiveLeavesUpdateSignal) error
```

Comment:
Nice update to this interface, I'd like to do a similar `ProcessBlockFinalazedSignal`